README.md

```diff
@@ -80,6 +80,33 @@ Examples:
 "tidalcycles.useBootFileInCurrentDirectory" : true
 ```
 
+### Hover / completion support for Tidal statements
+
+This extension implements some code support features for Tidal specific
+statements, providing code completion and hover information.
+
+You can set the detail level of the provided information through two
+configuration settings, one for the `hover` feature and one for `completion`:
+
+```
+"tidalcycles.codehelp.hover.level": "FULL"
+"tidalcycles.codehelp.completion.level" : "FULL"
+```
+
+Available levels for bothh options are:
```

Comment:
If you don't mind correcting the spelling for "both", then I will merge right away!

Reply:
Good catch, my lysdexia [sic] sometimes shows :) I've taken the liberty in 2509196 to correct another spelling issue, though unrelated to this pull request, as well. Hope that's OK.